spark-indexer/src/test/resources/2020_02/proteome/proteome.xsd

```diff
@@ -0,0 +1,254 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do We need this XSD file? test application.properties is pointing to source main proteome xsd. I was not able to find a place in the tests that we use this.

Reply:
the tests refer from the test resources, so we need it.